FrontEnd/src/Appointment/AppointmentPage.jsx

```diff
@@ -75,67 +89,109 @@ function AppointmentPage({ userData }) {
     upcomingAppointmentData.length === 0
   ) {
     return (
-      <div className="appointmentPage">
-        <h2>No appointments found.</h2>
+      <div className={`appointmentPage ${theme}`}>
+        <div className="appointments">
+          <h2>No appointments found.</h2>
+        </div>
       </div>
     );
   }
   return (
-    <div className="appointmentPage">
-      <h2>Pending Appointments</h2>
-      {pendingAppointmentData.map((appointment, index) => (
-        <div key={index}>
-          <p>{appointment.appointmentTitle}</p>
-          {appointment.vendorUID === userID && (
-            <div>
-              <button
-                onClick={() =>
-                  handleAcceptedAppointment(
-                    appointment.customerUID,
-                    appointment.vendorUID,
-                    appointment.docID
-                  )
-                }
-              >
-                Accept
+    <div className={`appointmentPage ${theme}`}>
+      <div className="appointments">
+        <h2>Pending Appointments</h2>
+        {pendingAppointmentData.map((appointment, index) => (
+          <div key={index} className="appointmentTab">
+            <div className="appointmentInfo">
+              {appointment.vendorUID === userID && (
+                <p className="appointmentUser">
+                  {appointment.customerUsername}
+                </p>
+              )}
+              {appointment.customerUID === userID && (
+                <p className="appointmentUser">{appointment.vendorUsername}</p>
+              )}
+              <p className="appointmentTitle">{appointment.appointmentTitle}</p>
+              <div className="appointmentDetail">
+                <i className="fa-solid fa-calendar"></i>
+                <p>{appointment.appointmentDate}</p>
+              </div>
+              <div className="appointmentDetail">
+                <i className="fa-solid fa-clock"></i>
+                <p>{appointment.appointmentTime}</p>
+              </div>
+            </div>
+            {appointment.vendorUID === userID && (
+              <div className="appointmentActions">
+                <button
+                  className="appointmentButtons"
+                  onClick={() =>
+                    handleAcceptedAppointment(
+                      appointment,
+                      appointment.customerUID,
+                      appointment.vendorUID,
+                      appointment.docID,
+                      3
```

Comment:
Can this type be saved into a reusable const in your utils? Because it's not clear what 3 refers to.